docs/source/experiment/webui.rst

```diff
@@ -0,0 +1,413 @@
+WebUI
+=====
+
+
+Q&A
+---
+
+There are many trials in the detail table but ``Default Metric`` chart is empty.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
remove `.` or add `?` at the end of each question?

Reply:
ok, remove `.`